pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-ssot-infoblox"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
I think this change is worth a minor bump, since it is adding new features.

Reply:
Good point. This is why we need to get semantic-release added into the project so I don't have to think of these things.

Reply:
I've corrected this.